apim/1.x/values.yaml

```diff
@@ -360,6 +360,9 @@ api:
   ingress:
     enabled: true
     path: /management
+    # Set enabled to true to use openshift-routes instead of kubernetes-ingress
+    route:
+      enabled: false
```

Comment:
You have defined a route flag per component (gateway, api, ui, ...).

I'm wondering if it would not be easier to manage an "openshift mode" with a single / global flag property ?

Also, to make things easier to understand, I would rename `route` for `openshift` to remove ambiguity.

WDYT?

Reply:
A single "openshift mode" global flag is in my opinion a bad idea, as people are using openshift differently some are using the routes openshift provides while other uses the stock ingress method provided by kubernetes. 

I'd rather have a couple of global openshift flags one for using openshift-routes instead of kubernetes-ingress and perhaps one for security contexts which should probably be added as another PR